### PR TITLE
update word from currently to may

### PR DESCRIPTION
**Pull request by robberfree**
Test on iOS 14.5:
Image component support that four border radius properties now.

### CODE REVIEW

**Review by Simek — COMMENTED** [review]
Hi @robberfree, thank you for the update, however I think this will need a little bit more of research and confirmation.

It would be helpful, if you can provide the PR, commit or changelog entry which indicated that support for those properties has been added to iOS (and if so, what are the exact requirements/circumstances to make the border radius work).

**Comment by robberfree** [comment]
@Simek good suggestions. But if you view the `Image style props` document here: https://reactnative.dev/docs/image-style-props#image-border-radius. You can see that the four `border radius` props are supported on **iOS** platform.

Another thing I check the history of the document: https://github.com/facebook/react-native-website/commits/master/docs/images.md. The sentences about `iOS Border Radius Styles` never have changed since first commit.

thanks.

cc: @hramos

**Comment by Simek** [comment]
@robberfree I have in mind the React Native core repository PRs and commits, or official changelog entries, not the docs.

The code should be the source of truth in this case.

File: docs/images.md [hunk 1]
```diff
@@ -218,7 +218,7 @@ Note that you must specify some width and height style attributes.
 
 ## iOS Border Radius Styles
 
-Please note that the following corner specific, border radius style properties are currently ignored by iOS's image component:
+Please note that the following corner specific, border radius style properties are may ignored by iOS's image component:
```
Comment (Simek) [inline docs/images.md:221]:
```suggestion
Please note that the following corner specific, border radius style properties might be ignored by iOS's image component:
```

Comment (Simek) [reply docs/images.md:221]: I'm going to merge the suggestion and land the PR, so it won'y become stale. 😉 

Also the duplicates start to appear:
* https://github.com/facebook/react-native-website/pull/2908

**Pull request merged**